tests/config/pytest_software.yml

```diff
@@ -0,0 +1,820 @@
+abacas:
+  - software/abacas/**
+  - tests/software/abacas/**
```

Comment:
This file has now been changed to `pytest_modules.yml` so you will need to use the latest version of tools or manually edit that file. This is why the tests aren't running.

Reply:
Yup, fixed!